client/webserver/site/src/js/forms.ts

```diff
@@ -87,8 +87,6 @@ export class NewWalletForm {
     // WalletConfigForm will set the global app variable.
     this.subform = new WalletConfigForm(page.walletSettings, true)
 
-    Doc.bind(this.subform.showOther, 'click', () => Doc.show(page.walletSettingsHeader))
```

Comment:
This is slightly too heavy handed to hide the load file icon because that's just a _part_ of the "Wallet Settings" header, which is still applicable when you expand the additional settings that seeded wallets still have.

```diff
@@ -336,6 +338,10 @@ export class NewWalletForm {
     if (this.subform.dynamicOpts.children.length) Doc.show(page.walletSettingsHeader)
     else Doc.hide(page.walletSettingsHeader)
 
+    // A seeded or token wallet is internal to the dex client ad as such does
+    // not have an external config file to select.
+    if (walletDef.seeded || Boolean(this.current.asset.token)) Doc.hide(this.subform.fileSelector)
+
     this.refresh()
     await this.loadDefaults()
   }
```